powertools-cloudformation/src/main/java/software/amazon/lambda/powertools/cloudformation/CloudFormationResponse.java

```diff
@@ -146,7 +164,7 @@ SdkHttpClient getClient() {
      */
     public HttpExecuteResponse send(CloudFormationCustomResourceEvent event,
                                     Context context) throws IOException, CustomResourceResponseException {
-        return send(event, context, null);
+        return send(event, context, Response.success(context.getLogGroupName()));
```

Comment:
physicalResourceId injection - and here

Reply:
@mriccia it's `public` - not sure we can change it in case a consumer of the powertools is using it?